### PR TITLE
Initial config initialization

### PR DESCRIPTION
**Pull request by lvfxx**
Implementation of #11

### CODE REVIEW

File: local/sdk-sriov/pkg/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+)
+
+// IsSriovCapable check if a pci device SR-IOV capable given its pci address
+func IsSriovCapable(pciAddr string) bool {
+	totalVfFilePath := filepath.Join(sysBusPci, pciAddr, totalVfFile)
+	if _, err := os.Stat(totalVfFilePath); err != nil {
+		return false
+	}
+	// sriov_totalvfs file exists -> sriov capable
+	return true
```
Comment (denis-tingajkin) [inline local/sdk-sriov/pkg/utils/utils.go:49]: this block can be converted in common local function and can be reused in IsSriovVF

File: local/sdk-sriov/pkg/utils/utils.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+	}
+	return true
+}
+
+// GetVFconfigured returns number of VF configured for a PF
+func GetVFconfigured(pf string) int {
+	configuredVfPath := filepath.Join(sysBusPci, pf, configuredVfFile)
+	vfs, err := ioutil.ReadFile(filepath.Clean(configuredVfPath))
+	if err != nil {
+		return 0
```
Comment (denis-tingajkin) [inline local/sdk-sriov/pkg/utils/utils.go:66]: Should we return an error here or 0 is invalid value?

File: local/sdk-sriov/pkg/utils/utils.go [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+	}
+	return numvfs
+}
+
+// DeviceExists validates PciAddr given as string and checks if it exists
+func DeviceExists(pciAddr string) error {
+	// Check system pci address
+
+	// sysbus pci address regex
+	var validLongID = regexp.MustCompile(`^0{4}:[0-9a-f]{2}:[0-9a-f]{2}.[0-7]{1}$`)
```
Comment (denis-tingajkin) [inline local/sdk-sriov/pkg/utils/utils.go:101]: let's use common local values and something like sync.Once to reduce regexp complication here.

File: local/sdk-sriov/pkg/utils/utils.go [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+	var validShortID = regexp.MustCompile(`^[0-9a-f]{2}:[0-9a-f]{2}.[0-7]{1}$`)
+
+	if validLongID.MatchString(pciAddr) {
+		return deviceExist(pciAddr)
+	} else if validShortID.MatchString(pciAddr) {
+		pciAddr = "0000:" + pciAddr // make short form to sysfs represtation
```
Comment (denis-tingajkin) [inline local/sdk-sriov/pkg/utils/utils.go:107]: What means `0000:`?

Comment (lvfxx) [reply local/sdk-sriov/pkg/utils/utils.go:107]: PCI addresses in the `/sys/bus/pci/devices/` are represented in the extended form of the [BDF notaion](https://wiki.xen.org/wiki/Bus:Device.Function_(BDF)_Notation). We allow user to specify PCI addresses also in short notation, adding in this case leading Domain zeros to get valid `/sys/bus/pci/devices/...` filepath.

Comment (denis-tingajkin) [reply local/sdk-sriov/pkg/utils/utils.go:107]: ok, let's move `0000:` it to const and add a comment with  BDF notation.

File: local/sdk-sriov/pkg/utils/utils.go (hunk 2 above)
Comment (lvfxx) [reply local/sdk-sriov/pkg/utils/utils.go:66]: I think we should return an arror, 0 is valid value

File: local/sdk-sriov/pkg/utils/utils.go (hunk 1 above)
Comment (lvfxx) [reply local/sdk-sriov/pkg/utils/utils.go:49]: Yes, it can, thanks

**Pull request closed without merging**